### PR TITLE
Do not use hook AdminStatsModules alias, use displayAdminStatsModules & Bump version to 2.0.4

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Do not use hook AdminStatsModules alias, use displayAdminStatsModules
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | n/d
| Possible impacts? | n/d

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

File: upgrade/upgrade-2.0.4.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+/**
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_2_0_4$module)
```
Comment (PierreRambaud) [inline upgrade/upgrade-2.0.4.php:27]:
```suggestion
function upgrade_module_2_0_4($module)
```

File: upgrade/index.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
+header('Last-Modified: '.gmdate('D, d M Y H:i:s').' GMT');
```
Comment (atomiix) [inline upgrade/index.php:28]:
```suggestion
header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
header('Last-Modified: ' . gmdate('D, d M Y H:i:s').' GMT');
```

Comment (PierreRambaud) [reply upgrade/index.php]: Seems like it's not enough :p

**Review by matthieu-rolland — COMMENTED** [review]
seems like the CI is blocked by a code style issue, this command will fix it:

`./vendor/friendsofphp/php-cs-fixer/php-cs-fixer fix upgrade/index.php`

then commit, push, and your PR will be fine 👍

File: upgrade/index.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
+header('Last-Modified: ' . gmdate('D, d M Y H:i:s').' GMT');
```
Comment (PierreRambaud) [inline upgrade/index.php:27]:
```suggestion
header('Last-Modified: ' . gmdate('D, d M Y H:i:s') . ' GMT');
```

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @PrestaEdit

**Pull request merged**